src/lib/otp_lib/capped_supervisor.ml

```diff
@@ -0,0 +1,46 @@
+open Core_kernel
+open Async_kernel
+open Pipe_lib
+open Strict_pipe
+
+type 'job t =
```

Comment:
can you call this `'input` or `'data` or `'a`? For me `'job` implies that the type variable is being replaced by a function itself, I was a bit confused by that while I was reading it.